assets/models/system/distilgpt2/spec.yaml

```diff
@@ -114,4 +119,20 @@ tags:
       Standard_ND96amsr_A100_v4,
       Standard_ND40rs_v2
     ]
+  finetune_compute_allow_list:
+    [
+      Standard_NV12s_v3,
+      Standard_NV24s_v3,
+      Standard_NV48s_v3,
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4,
```

Comment:
there will be no issues with this additional ',' at the end in yaml files right? Can once check by loading the yaml file? (This is there for all   finetune_compute_allow_list)

Reply:
there will be no issues, i have checked it.